src/include/function/gds/gds_utils.h

```diff
@@ -9,9 +9,12 @@ namespace function {
 
 class KUZU_API GDSUtils {
 public:
-    static void runFrontiersUntilConvergence(processor::ExecutionContext* context,
-        GDSComputeState& rjCompState, graph::Graph* graph, common::ExtendDirection extendDirection,
-        uint64_t maxIteration, const std::string& propertyToScan = "");
+    static void runGDS(processor::ExecutionContext* context, GDSComputeState& compState,
```

Comment:
I think `runFrontiersUntilConvergence` was more accurate than `runGDS`. This function should be part of the execution when we run a GDS algorithm, no?